src/region_growing/abstract.py

```diff
@@ -0,0 +1,39 @@
+"""Data Fuser Abstract Base Class"""
+
+from abc import ABC, abstractmethod
+
+
+class AbstractRegionGrowing(ABC):
+    """
+    Data Fuser abstract base class for automatic labelling point clouds.
+
+    Parameters
+    ----------
+    label : int
+        Class label to use for this fuser.
+    """
+
+    def __init__(self, label):
+        self.label = label
+        super().__init__()
+
+    @abstractmethod
+    def get_label_mask(self, points):
```

Comment:
In de FusionPipeline worden er hier meer argumenten meegegeven, misschien moeten die dan ook in de abstract class?